setup.cfg

```diff
@@ -121,7 +121,6 @@ webdhfs_kerberos = requests-kerberos==0.14.0
 terraform = tpi[ssh]>=2.1.0
 tests =
     %(terraform)s
-    wheel==0.37.1
```

Comment:
`wheel` is a build-time dependency for us. Using with `pip install -e ".[dev]"` will not install wheel-packages immediately, it only takes effect afterwards.